Add FAQs to Python step.parallel page (#795)
inngest/website: pages/docs/reference/python/steps/parallel.mdx

        )
    )
```

> ## Frequently asked questions

Reviewer: ```suggestion
## Frequently Asked Questions
```

### Do parallel steps work if I don't use `async` functions?

> Yes, parallel steps work with both `async` and non-`async` functions. Since our execution model uses a separate HTTP request for each step, threaded HTTP frameworks (e.g. Flask) will create a separate thread for each step.

Reviewer: ```suggestion
Yes, parallel steps work with both `async` and non-`async` functions. Since our execution model uses a separate HTTP request for each step, threaded HTTP frameworks (for example, Flask) will create a separate thread for each step.
```

### Can I use `asyncio.gather` instead of `step.parallel`?

> No, `asyncio.gather` will not work as expected. Our execution model necessitates a control flow interruption when we encounter a `step` method, but we don't currently have a way to make that work with `asyncio.gather`.

Reviewer: ```suggestion
No, `asyncio.gather` will not work as expected. Inngest's execution model necessitates a control flow interruption when it encounters a `step` method, but currently that does not work with `asyncio.gather`.
```

Less personal
